python/instrumentation/openinference-instrumentation-litellm/src/openinference/instrumentation/litellm/__init__.py

```diff
@@ -51,10 +105,16 @@ def _instrument_func_type_completion(span: trace_api.Span, kwargs: Dict[str, Any
     _set_span_attribute(span, SpanAttributes.LLM_MODEL_NAME, kwargs.get("model", "unknown_model"))
 
     if messages := kwargs.get("messages"):
-        _set_span_attribute(span, SpanAttributes.INPUT_VALUE, str(messages[0].get("content")))
-        for i, obj in enumerate(messages):
-            for key, value in obj.items():
-                _set_span_attribute(span, f"input.messages.{i}.{key}", value)
+        if content := messages[0].get("content"):
+            if isinstance(content, str):
+                _set_span_attribute(span, SpanAttributes.INPUT_VALUE, str(content))
+            elif is_iterable_of(content, dict):
+                _set_span_attribute(span, SpanAttributes.INPUT_VALUE, str(messages))
```

Comment:
I don't understand this, if the content is a dict, we set all the messages as the input value?

Reply:
Resolved with @RogerHYang and @fjcasti1- for now just setting input value to be `json.dumps(messages)`